src/app/txRunner.js

```diff
@@ -65,25 +65,26 @@ TxRunner.prototype.execute = function () {
         })
       })
     } else {
-      self.web3.eth.estimateGas(tx, function (err, resp) {
+      self.web3.eth.estimateGas(tx, function (err, gasEstimation) {
```

Comment:
Can you please split this into two commits? (changing `resp` to `gasEstimation` and the actual change)